extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -33,6 +33,8 @@ import {
   installUrlReplacementsService,
 } from '../../../../src/service/url-replacements-impl';
 import * as sinon from 'sinon';
+import {installStorageService} from '../../../../src/service/storage-impl';
+import '../../../amp-analytics/0.1/amp-analytics';
 
```

Comment:
I don't think it is needed. Just forgot to delete this probably. Will test and let you know.